src/streamlink/plugins/btv.py

```diff
@@ -60,8 +61,10 @@ def _get_streams(self):
         if stream_url == "geo_blocked_stream":
             log.error("The content is not available in your region")
             return
-
-        return HLSStream.parse_variant_playlist(self.session, stream_url)
+        
+        hls_url = update_scheme("hls://https://", stream_url, force=False)
```

Comment:
`update_scheme()` doesn't make any sense here and needs to be removed:

1. `hls://` is the explicit scheme of Streamlink's HLS **plugin**, so that direct HLS playlist URL inputs can be handled whose path don't end with `.m3u8`.
   See https://github.com/streamlink/streamlink/blob/6.4.2/src/streamlink/plugins/hls.py#L13-L15
2. `update_scheme()` does only apply the scheme to the stream URL if it's missing, which is not the case here, hence the reason why you didn't notice any issues with `hls://`. If you'd actually add `hls://` to the stream URL, then `requests` couldn't find a matching adapter for this URL.